sidebars.ts

```diff
@@ -258,6 +258,7 @@ module.exports = {
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/akamai-siem-api-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source',
+                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source',
```

Comment:
If this article is in beta, do we want it to appear in the left nav?

Reply:
I have hidden this line for now, thanks for identifying this John. There was a confusion on weather to release this source as beta or not, so missed it here.